src/ApiKitClient.ts

```diff
@@ -8,10 +8,12 @@ export class ApiKitClient {
   private static instance: AxiosInstance;
   private static authTokenCallback: AuthTokenCallback;
   private static unauthorizationCallback: UnauthorizationCallback;
+  private static useAuth: boolean = true;
 
-  public static initialize(baseURL: string, authTokenCallback: AuthTokenCallback, unauthorizationCallback?: UnauthorizationCallback): void {
+  public static initialize(baseURL: string, authTokenCallback: AuthTokenCallback, unauthorizationCallback?: UnauthorizationCallback, useAuth?: boolean): void {
```

Comment:
`authTokenCallback: AuthTokenCallback` this should be optional too, why require AuthTokenCallback for unauthorized API

Reply:
So in case we want to use unauthed API call we have to create 2 API clients in the project ?
why can't we make it 1 API client per project and the authed boolean to be inside the functions and not initiation
ex : 
for unauthed calls 
```
const useAuth = false;
Api.get(params, useAuth);
```
for authed called
```
Api.get(params) 
```
so useAuth is true unless it is passed false

Reply:
Thanks, you're correct!
I missed totally to revisit the `authTokenCallback` param...